core-services/src/main/java/org/zalando/nakadi/service/publishing/AvroEventPublisher.java

```diff
@@ -65,13 +63,13 @@ public void publishAvro(final String etName,
                             final GenericRecord event) {
         try {
             final GenericRecord metadata = new GenericData.Record(avroSchema.getMetadataSchema());
-            final DateTime dateTime = new DateTime(DateTimeZone.UTC);
-            metadata.put("occurred_at", dateTime);
+            final long now = System.currentTimeMillis();
+            metadata.put("occurred_at", now);
```

Comment:
Wondering why wasn't it failing previously, if the data type didn't match?

Reply:
it should be failing ...